server/channels/api4/channel_test.go

```diff
@@ -2478,9 +2478,10 @@ func TestViewChannel(t *testing.T) {
 	CheckBadRequestStatus(t, resp)
 
 	view.ChannelId = "correctlysizedjunkdddfdfdf"
-	_, resp, err = client.ViewChannel(context.Background(), th.BasicUser.Id, view)
-	require.Error(t, err)
-	CheckBadRequestStatus(t, resp)
+	viewResult, _, err := client.ViewChannel(context.Background(), th.BasicUser.Id, view)
+	require.NoError(t, err)
+	require.Len(t, viewResult.LastViewedAtTimes, 0)
+
```

Comment:
Oh, what caused this change? It's exactly the opposite, right?

Reply:
So... since we are always using "view multiple" I think we don't want to fail if there is some id wrong. So that id will just be skipped, and the `LastViewedAtTimes` will just not have that id present. Happy to think about any other possibility here.

Reply:
Ah, got it! Yeah, it makes sense, thanks for the clarification :)